enterprise/aibridged/aibridged_test.go

```diff
@@ -17,6 +17,7 @@ import (
 
 	"cdr.dev/slog/sloggers/slogtest"
 	"github.com/coder/aibridge"
+	coderaibridge "github.com/coder/coder/v2/coderd/aibridge"
```

Comment:
Nit: the common pattern is to name this like `agplaibridge`.
See `enterprise/cli/root.go`.